sens.toit

```diff
@@ -55,7 +56,7 @@ main:
 // trusted root certificate in Chrome at chrome://settings/certificates (the
 // Authorities tab) and export it to a text file.
 
-GLOBALSIGN_ROOT ::= net.Certificate.parse """\
+GLOBALSIGN_ROOT ::= net.Certificate.parse """
```

Comment:
In theory the '\' shouldn't be necessary. There is a tiny tiny chance I'm wrong, in which case just add it back.